iModelCore/BeSQLite/CloudSqlite.cpp

```diff
@@ -25,7 +25,8 @@ Utf8String Db::OpenParams::SetFromContainer(Utf8CP dbName, CloudContainerP conta
     m_skipFileCheck = true;
     m_fromContainer = true;
     AddQueryParam(Utf8PrintfString("vfs=%s", container->m_cache->m_name.c_str()).c_str());
-    AddQueryParam(Utf8PrintfString("cloudSqliteLogId=%s", container->m_cloudSqliteLogId.c_str()).c_str()); 
+    if (!container->m_logId.empty())

```

Comment:
@nick4598 note this change. I think it's right, but please verify (if the `logId` is empty, don't bother calling PRAGMA bcv_client).

Reply:
Yep, looks good.